plugins/resources/src/compound.cpp

```diff
@@ -1626,6 +1626,26 @@ irods::error open_for_prefer_archive_policy(
         return SUCCESS();
     }
 
+    std::string cache_name;
+    if (const auto err = _ctx.prop_map().get<std::string>(CACHE_CONTEXT_TYPE, cache_name); !err.ok()) {
+        return PASS(err);
+    }
+
+    try {
+        // Set the vote for the replica in the cache resource to be the same as the vote for the replica in the archive
+        // because the replica in the cache is going to be overwritten by the replica in the archive anyway. The vote
+        // must be set for the cache replica because the user may have requested the replica in the cache replica
```

Comment:
"replica in the cache replica" -> "replica in the cache"

Reply:
Done